lib/Model/index.ts

```diff
@@ -221,7 +221,7 @@ interface ModelBatchDeleteSettings {
 }
 
 // Model represents one DynamoDB table
-export class Model<T extends DocumentCarrier> {
+export class Model<T extends DocumentCarrier, K = InputKey> {
```

Comment:
So. `InputKey` by the naming was meant to be a specific key. I think maybe we want to rename this, since I know it was defined as a really broad thing. Not sure what makes sense here tho.